eo-phi-normalizer/Setup.hs

```diff
@@ -1,3 +1,25 @@
+-- The MIT License (MIT)
```

Comment:
Fourmolu wants to reduce blank lines in the license text. Let's try to preserve these blank lines. 

Add `{- FOURMOLU_DISABLE -}` before the first line of license in `.hs` files. 
Add `{- FOURMOLU_ENABLE -}` after the last line of license in `.hs` files.